SwiftMonkeyPaws/Sources/Extensions/UIColor+MonkeyPaws.swift

```diff
@@ -0,0 +1,26 @@
+//
```

Comment:
We now have a quite weird file structure on disk:

![bildschirmfoto 2018-02-11 um 23 52 22](https://user-images.githubusercontent.com/1027098/36079746-bfcb4f5a-0f86-11e8-97c0-f8f73845bec9.png)

The source files are in `SwiftMonkeyPaws` and in `SwiftMonkeyPaws/Sources/Extensions`. We have 2 options:
1. Move `MonkeyPawDrawer.swift`, `MonkeyPaws.swift` and `Package.swift` to the `Sources` folder and update the `s.source_files` in the `SwiftMonkeyPaws.podspec`
2. Move all files from the `SwiftMonkeyPaws/Sources/Extensions` to `SwiftMonkeyPaws` folder